Lesson9/anna/index.html

```diff
@@ -6,6 +6,395 @@
+                  alt="FreeCodeCamp logo"
+                />
+                </a>
+            </div>
+          </div>
+          <div class="text-info-container">
+            <div class="text-info-row">
+              <a href="https://medium.com/@adri_zag">Adrienne Zaganelli</a>
+              <span class="separator--dot">·</span>
+              <a class="link--success" href="https://medium.com/m/signin">Follow</a>
+            </div>
+            <div class="text-info-row">
+              <span class="text-secondary">Published in&nbsp;</span>
+              <a class="info-link" href="https://medium.com/free-code-camp">We’ve moved to freeCodeCamp.org/news
+                <span class="separator--dot">·</span>
+              </a>
+              <span class="text-secondary">3 min read</span>
+              <span class="separator--dot"> · </span>
+              <span class="text-secondary">Aug 12, 2018</span>
+            </div>
+          </div>
+          
+        </div>
+        <div class="article-actions-container">
+          <div class="article-actions-left">
+            <div class="first">
+            <svg 
```

Comment:
It's ok to use SVGs here but the code would be cleaner if you just find a similar icon from an icon library (e.g. font-awesome) as SVG code is long an hard to read. :)